README.md

```diff
@@ -1,4 +1,4 @@
-# Google Software Product Sprint
+# My Portfolio
 
 This repo contains the projects you'll work on throughout SPS.
```

Comment:
Change this line to describe that it's your repo, i.e. something like "the projects I'll work on"...